apis/python/src/tiledbsoma/_collection.py

```diff
@@ -509,9 +509,49 @@ class Collection(
     CollectionBase[CollectionElementType], somacore.Collection[CollectionElementType]
 ):
     """
-    A persistent collection of SOMA objects, mapping string keys to any SOMA object.
+    ``Collection`` is a persistent collection of named SOMA objects, stored as
+    a mapping of string keys and SOMA object values. Values may be any
+    persistent ``tiledbsoma`` object, including ``DataFrame``,
+    ``SparseNDArray``, ``DenseNDArray``, ``Experiment``, ``Measurement``,
+    or another ``Collection``. A ``Collection`` refers to elements by a
+    per-element URI. A ``Collection`` may store its reference to an
+    element by absolute URI or relative URI.
 
     [lifecycle: experimental]
+
+    Examples:
+    ---------
+    >>> import tiledbsoma
+    >>> import pyarrow as pa
+    >>> import numpy as np
+    >>> # create a collection and add a (10,10) dense matrix to it
+    ... with tiledbsoma.Collection.create("./test_collection") as my_collection:
+    ...     with my_collection.add_new_dense_ndarray(
+    ...         "my_dense_ndarray", type=pa.int32(), shape=(10, 10)
+    ...     ) as my_dense_ndarray:
+    ...         data = pa.Tensor.from_numpy(np.eye(10, 10, dtype=np.int32))
+    ...         my_dense_ndarray.write((slice(None), slice(None)), data)
```

Comment:
reader has to parse a lot of less-relevant code just to see the Collection.create() and open() in action.

Reply:
True, but I wanted an example to show how to use it.

Can you please propose an edit or alternative? I'm quite open to changing it, but don't have an idea

Reply:
I suppose we could just add the empty ndarray or a child collection.  But this is really minor.  Maybe a comment after the opening `with` line: `# Add a collection member, such as a dense array` just to explain the rest is arbitrary code in the overall example.

Reply:
OK, added comments.